Source/ASNetworkImageNode.h

```diff
@@ -133,6 +133,13 @@ NS_ASSUME_NONNULL_BEGIN
  */
 @property (readonly) CGFloat renderedImageQuality;
 
+/**
+ * Download progress of the current image.
+ * When downloading a network image, this value would be updated to track download progress (value between 0 and 1)
+ * If the URL is unset, this is 1 if defaultImage or image is set to non-nil.
```

Comment:
> If the URL is unset, this is 1 if defaultImage or image is set to non-nil.

This line is not true for download progress because the progress should be 0 (set [here](https://github.com/TextureGroup/Texture/pull/1489/files#diff-652a444aea8e41da45fd4ba883780cccR212)).